project/profiles/capacity4more/modules/c4m/user/c4m_user/c4m_user.module

```diff
@@ -315,6 +315,11 @@ function c4m_user_action_info() {
  * Set's the message subject and body as configured.
  */
 function c4m_user_mail($key, &$message, $params) {
+
+  if ($key == 'user_contact_mail') {
```

Comment:
@anvmn 
Please add an elaborated comment explaining why we return early over here,
thanks